core/build.gradle

```diff
@@ -121,6 +121,7 @@ dependencies {
     testImplementation "org.camunda.community.mockito:camunda-platform-7-mockito:${camundaMockitoVersion}"
     testImplementation "org.springframework.security:spring-security-test"
     testImplementation "org.junit.jupiter:junit-jupiter-engine"
+    testImplementation "org.codehaus.groovy:groovy-all:3.0.20"
```

Comment:
Shouln't the version be in the `gradle.properties` file?

Reply:
Moved the version to `gradle.properties` and updated to the latest version